zipline/finance/performance/period.py

```diff
@@ -207,15 +225,34 @@ def adjust_cash(self, amount):
     def adjust_field(self, field, value):
         setattr(self, field, value)
 
+    def _get_payout_total(self, positions):
+        payouts = []
+        for asset, old_price in iteritems(self._payout_last_sale_prices):
+            pos = positions[asset]
+            price = pos.last_sale_price
```

Comment:
For my clarity --- at time T:
pos.last_sale_price is the price of 'asset' at time T
old_price is the last price for which a payout occurred (at time T-N)